plugins/utils/instant.nix

```diff
@@ -0,0 +1,104 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+} @ args:
+with lib; let
+  helpers = import ../helpers.nix args;
+in {
+  options.plugins.instant = {
+    enable = mkEnableOption "instant.nvim";
+
+    package = helpers.mkPackageOption "instant" pkgs.vimPlugins.instant-nvim;
+
+    username = helpers.mkNullOrOption types.str "Username";
```

Comment:
Would it be possible to add a warning if this option is not explicilty set to null ? (as this options is required by the plugin if I understood correctly)

Reply:
Yes, good point. It is better now.